common/org.testeditor.dsl.common/src/org/testeditor/dsl/common/ide/util/FileUtils.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ *******************************************************************************/
+package org.testeditor.dsl.common.ide.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.FileVisitor;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.attribute.BasicFileAttributes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Util class for file operations.
+ *
+ */
+public final class FileUtils {
+
+	/**
+	 * 
+	 */
+	private FileUtils() {
+		// do nothing
+	}
+
+	private static Logger logger = LoggerFactory.getLogger(FileUtils.class);
+
+	/**
+	 * copies the directories.
+	 * 
+	 * @param src
+	 *            source-directory
+	 * @param dest
+	 *            destination-directory
+	 * @throws IOException
+	 *             IOException
+	 */
+	public static void copyFolder(File src, File dest) throws IOException {
```

Comment:
Can't we use a library for this?
E.g. http://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/FileUtils.html#copyDirectory%28java.io.File,%20java.io.File%29